src/load_data.py

```diff
@@ -0,0 +1,9 @@
+import boto3
+
+def load_data(args): 
```

Comment:
Docstrings documentation can be added to help understand what this function does.